File: php/examples/basic.php
<?php

require __DIR__ . '/../vendor/autoload.php';

use CodeBridge\BridgeConfig;
use CodeBridge\Client;

$url = getenv('CODE_BRIDGE_URL') ?: 'ws://localhost:9877';
$secret = getenv('CODE_BRIDGE_SECRET') ?: 'dev-secret';

$client = new Client(new BridgeConfig($url, $secret, 'php-example'));
$client->start();
$client->sendConsole('hello from php');
$client->sendError('sample error');
$client->stop();

File: ruby/examples/basic.rb
#!/usr/bin/env ruby
require_relative '../lib/code_bridge_client'

url = ENV.fetch('CODE_BRIDGE_URL', 'ws://localhost:9877')
secret = ENV.fetch('CODE_BRIDGE_SECRET', 'dev-secret')

client = CodeBridge::Client.new(url: url, secret: secret, project_id: 'ruby-example')
client.start
client.send_console('hello from ruby')
client.send_error('sample error')
sleep 0.5
client.stop
